input/pagecontent/index.md

```diff
@@ -448,14 +419,15 @@ In this section, we will walk through a realistic example of FSH, line by line.
 * Line 6 gives an id for this profile. The id is used to create the globally unique URL for the profile. The URL is composed of the IG’s canonical URL, the instance type (always `StructureDefinition` for profiles), and the profile’s id.
 * Line 7 is a human-readable title for the profile.
 * Line 8 is the description that will appear in the IG on the profile's page.
-* Line 9 is the start of the rule section of the profile. It uses [caret syntax](reference.html#structuredefinition-escape-paths) to set the status attribute in the StructureDefinition produced for this profile.
+* Line 9 is the start of the rule section of the profile. It uses [caret syntax](reference.html#structuredefinition-escape-paths) to set the status attribute in the SD produced for this profile.
 * Line 10 adds an extension to the profile using the standalone extension, `EvidenceType`, gives it the local name `evidenceType`, and assigns the cardinality 0..*. _EvidenceType is defined on line 30._
 * Line 11 binds the valueCodeableConcept of the evidenceType extension to a value set named CancerDiseaseStatusEvidenceTypeVS with a required binding strength. _CancerDiseaseStatusEvidenceTypeVS is defined on line 46._
 * Line 12 designates a list of elements (inherited from Observation) as must-support.
 * Lines 13 to 20 constrain the cardinality of some inherited elements. FSH does not support setting the cardinality of a multiple items at a time, so these must be separate statements.
 * Lines 21 and 22 restrict the choice of resource types for two elements that refer to other resources.
 * Line 23 fixes the value of the code attribute to a specific LOINC code, using an alias for the code system defined on line 1.
```

Comment:
In line 23, consider putting the display as a comment -- since I think we consider that best practice.  Unless you think it's too complicated to describe why we do that in this section...

Reply:
We had a specific ballot comment to take out the comment and the explanation; the voter found it confusing and unnecessary.